pulsar-client-cpp/wireshark/pulsarDissector.cc

```diff
@@ -153,18 +230,15 @@ struct MessageIdComparator {
 
 struct RequestData {
     uint32_t requestFrame;
-    nstime_t requestTimestamp;
+    nstime_t requestTimestamp{};
```

Comment:
Why do you add `{}` here? The same question applies for other places and I won't comment again.

Reply:
Removed.